zulipterminal/ui_tools/boxes.py

```diff
@@ -578,6 +582,14 @@ def need_recipient_header(self) -> bool:
         else:
             raise RuntimeError("Invalid message type")
 
+    def _set_topic_links(self) -> None:
+        if 'topic_links' not in self.message.keys():
+            return
+        for link in self.message['topic_links']:
+            self.topic_links[link] = (
+                'Link', len(self.topic_links) + 1, True
```

Comment:
Not having the link text available is a little frustrating :frowning_face:

Perhaps the backend could be improved in this regard, as at least for linkifiers like we regularly use, we know there is "link text", like `#T867` here.

For now could we maybe special-case the view for empty link text and put it on one line?